types/comtac/CM-2/timer.schema.json

```diff
@@ -3,5 +3,11 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "title": "Timer",
   "type": "object",
-  "properties": {}
+  "properties": {
+    "status": {
```

Comment:
better call this timedUplink, when is this created?

Reply:
On a standart send intervals